.bcr/presubmit.yml

```diff
@@ -6,7 +6,8 @@ bcr_test_module:
     # File "C:/b/eznkjcld/external/rules_oci~override/oci/repositories.bzl", line 100, column 64, in _stucture_test_repo_impl
     # integrity = ST_VERSIONS[repository_ctx.attr.st_version][platform],
     # Error: key "windows-amd64" not found in dictionary
-    platform: ["debian10", "macos", "ubuntu2004"]
+    # TODO(#603): add macos
+    platform: ["debian10", "ubuntu2004"]
```

Comment:
I think we could just make a correct fix now instead - the `test_targets` below should have a different result for one platform. I can take it if you like.

Reply:
that'd be better. let me close this PR and cut a release so you can iterate on BCR until you get it green.